code/datums/intercept_text.dm

```diff
@@ -96,7 +96,7 @@
 	var/prob_right_dude = rand(prob_correct_person_lower, prob_correct_person_higher)
 	var/prob_right_job = rand(prob_correct_job_lower, prob_correct_job_higher)
 	if(prob(prob_right_job))
-		if (correct_mob)
+		if (correct_mob && correct_mob:assigned_role != "MODE")
 			traitor_job = correct_mob:assigned_role
```

Comment:
This check should be with the `prob` above as this will make `traitor_job` completely blank instead of filling in a different job.